Add support importing firewall rules

`terraform import module.unifi-fw-rule-2017.unifi_firewall_rule.ufr 5fb394c330905f01c4df9c98                     ✔  14s    09:35:20   
module.unifi-fw-rule-2017.unifi_firewall_rule.ufr: Importing from ID "5fb394c330905f01c4df9c98"...
module.unifi-fw-rule-2017.unifi_firewall_rule.ufr: Import prepared!
  Prepared unifi_firewall_rule for import
module.unifi-fw-rule-2017.unifi_firewall_rule.ufr: Refreshing state... [id=5fb394c330905f01c4df9c98]

Error: unable to decode body: GET s/default/rest/firewallrule/5fb394c330905f01c4df9c98 json: cannot unmarshal string into Go struct field FirewallRule.data.rule_index of type int`

I'm running: 6.0.28 and I have USG-3P. 

`2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: HTTP/1.1 200 
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: Transfer-Encoding: chunked
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: Access-Control-Allow-Credentials: true
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: Access-Control-Expose-Headers: Access-Control-Allow-Origin,Access-Control-Allow-Credentials
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: Content-Type: application/json;charset=UTF-8
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: Date: Thu, 19 Nov 2020 18:22:06 GMT
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: Vary: accept-encoding,origin,accept-encoding
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: X-Frame-Options: DENY
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: 
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: 256
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: {
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:  "meta": {
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:   "rc": "ok"
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:  },
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:  "data": [
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:   {
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "_id": "5fb394c330905f01c4df9c98",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "action": "reject",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "enabled": false,
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "dst_address": "",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "dst_firewallgroup_ids": [],
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "dst_networkconf_type": "NETv4",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "icmp_typename": "",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "ipsec": "",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "logging": false,
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "name": "TEST-TEST-12",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "protocol": "all",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "protocol_match_excepted": false,
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "ruleset": "LAN_IN",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "src_firewallgroup_ids": [],
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "src_address": "",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "src_mac_address": "",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "src_networkconf_type": "NETv4",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "state_established": false,
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "state_invalid": false,
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "state_new": false,
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "state_related": false,
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "dst_networkconf_id": "",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "src_networkconf_id": "",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "rule_index": "2017",
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:    "site_id": "5d00c6f61a0544241b05f36c"
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:   }
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0:  ]
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: }
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: 0
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: 
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: 
2020-11-19T18:22:06.591Z [DEBUG] plugin.terraform-provider-unifi_v0.18.0: -----------------------------------------------------
2020/11/19 18:22:06 [ERROR] eval: *terraform.EvalRefresh, err: unable to decode body: GET s/default/rest/firewallrule/5fb394c330905f01c4df9c98 json: cannot unmarshal string into Go struct field FirewallRule.data.rule_index of type int
2020/11/19 18:22:06 [ERROR] eval: *terraform.EvalSequence, err: unable to decode body: GET s/default/rest/firewallrule/5fb394c330905f01c4df9c98 json: cannot unmarshal string into Go struct field FirewallRule.data.rule_index of type int
2020/11/19 18:22:06 [TRACE] [walkImport] Exiting eval tree: import module.unifi-fw-rule-2017.unifi_firewall_rule.ufr result
2020/11/19 18:22:06 [TRACE] vertex "import module.unifi-fw-rule-2017.unifi_firewall_rule.ufr result": visit complete
2020/11/19 18:22:06 [TRACE] vertex "module.unifi-fw-rule-2017.unifi_firewall_rule.ufr (import id \"5fb394c330905f01c4df9c98\")": dynamic subgraph encountered errors
2020/11/19 18:22:06 [TRACE] vertex "module.unifi-fw-rule-2017.unifi_firewall_rule.ufr (import id \"5fb394c330905f01c4df9c98\")": visit complete
2020/11/19 18:22:06 [TRACE] dag/walk: upstream of "module.unifi-fw-rule-2017 (close)" errored, so skipping
2020/11/19 18:22:06 [TRACE] dag/walk: upstream of "provider[\"registry.terraform.io/paultyng/unifi\"] (close)" errored, so skipping
2020/11/19 18:22:06 [TRACE] dag/walk: upstream of "root" errored, so skipping`

--- a/go.mod
+++ b/go.mod
@@ -27,7 +27,7 @@ require (
 	github.com/mattn/go-colorable v0.1.8 // indirect
 	github.com/mitchellh/cli v1.1.2 // indirect
 	github.com/mitchellh/go-testing-interface v1.14.1 // indirect
-	github.com/paultyng/go-unifi v1.10.0
+	github.com/paultyng/go-unifi v1.10.2
 	github.com/posener/complete v1.2.1 // indirect
 	github.com/vmihailenco/msgpack v4.0.4+incompatible // indirect
 	github.com/zclconf/go-cty v1.6.1 // indirect

